tests/test_embed/test_interpreter.cpp

```diff
@@ -323,3 +323,32 @@ TEST_CASE("sys.argv gets initialized properly") {
     }
     py::initialize_interpreter();
 }
+
+class Counter {
+public:
+    Counter() { cnt_++; }
+    Counter(const Counter &) { cnt_++; }
+    Counter(Counter &&) { cnt_++; }
+    ~Counter() { cnt_--; }
+    static int get_cnt() { return cnt_; }
```

Comment:
```suggestion
    static int get_cnt() const { return cnt_; }
```

Reply:
error: static member function `static int Counter::get_cnt()` cannot have cv-qualifier

Reply:
Ah, sorry forgot it was static.